torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -9,6 +10,7 @@
     Callable,
     cast,
     Dict,
+    Generator,
     List,
     Optional,
     Generator,
```

Comment:
lint error is real

Reply:
Good catch. My VSCode Python extension broke starting yesterday, and I was finally able to fix it by reverting to an older version. :sweat_smile: